app/models/website.rb

```diff
@@ -13,4 +13,20 @@ def long_url_must_begin_with_protocol
     end
   end
 
+  def increase_hit_count
+    update_attributes(hit_count: hit_count + 1)
+  end
+
+  def self.ranked_by_hit_count
+    order('hit_count DESC')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.